src/apps/chifra/pkg/types/types_traceaction.go

```diff
@@ -62,7 +62,7 @@ func (s *SimpleTraceAction) Model(showHidden bool, format string, extraOptions m
 		if s.Value.String() != "0" {
 			model["value"] = s.Value.String()
 		}
-		if len(s.Init) > 0 {
+		if extraOptions["tracesTransactionsFormat"] != true && len(s.Init) > 0 {
```

Comment:
I'm a bit confused by this. In general, when there's a command line option to a tool (such as `chifra transactions --traces`) I would have used `traces` instead of `tracesTransactionsFormat`. It just causes less cognitive load since there is no "new word" to understand. I will probably change this once it's merged.

Reply:
Yes. This whole thing is confusing. I'm certain we're going to revisit the model code. One thing I've had in mind is that each different "version" of the data should actually have its own model.